### DISCUSSION TITLE
What is the last TB version which works with Java v8? Where to download?

### DISCUSSION THREAD

**Discussion by pstein** [mytourbook/mytourbook#1127] [General]
What is the last MyTourbook  version which works with Java v8?

**Comment by FJBDev** [comment]
The last version working with Java 8 would be [20.8](https://mytourbook.sourceforge.io/mytourbook/index.php/history/history-2020#v20.8.0) as Java 11 was introduced in [20.12](https://mytourbook.sourceforge.io/mytourbook/index.php/history/history-2020#v20.12.0)
